components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/resources/identity-governance.yaml

```diff
@@ -710,6 +714,18 @@ components:
           type: string
           description: Property description.
           example: Enable account suspend notifications.
+        meta:
+          type: object
+          description: Meta data properties
```

Comment:
Lets update as `Meta Data related to each property`

Reply:
Fixed in commit 37c3b01b50911acef42cdeb84eb9d2405b1c1c71